src/nupic/frameworks/opf/modelfactory.py

```diff
@@ -80,13 +81,17 @@ def create(modelConfig, logLevel=logging.ERROR):
     return modelClass(**modelConfig['modelParams'])
 
   @staticmethod
-  def loadFromCheckpoint(savedModelDir, newSerialization=False):
-    """ Load saved model.
+  def loadFromCheckpoint(savedModelDir):
+    """Load saved model.
+
     @param savedModelDir (string)
-           Directory of where the experiment is to be or was saved
+        Directory of where the experiment is to be or was saved
     @returns (nupic.frameworks.opf.model.Model) The loaded model instance.
     """
-    if newSerialization:
+    if os.path.exists(os.path.join(savedModelDir, "model.data")):
```

Comment:
Yeah maybe. What do you think? And would we need two functions then for the two checkpoint formats?